src/main/resources/static/openapi/EducationAndWorkPlanAPI.yml

```diff
@@ -821,6 +821,10 @@ components:
         archiveReasonOther:
           type: string
           description: Describes the reason for archiving if 'OTHER' is selected, it is mandatory in this scenario.
+        archiveNote:
```

Comment:
We normally bump the version of the swagger spec when we make any changes (semver)
This is a non breaking change, so perhaps just a patch bump?
(line 4 of the swagger spec)